plugin-packs/postcss-preset-env/src/lib/browsers-with-supports-stats.mjs

```diff
@@ -0,0 +1,16 @@
+// Matches the browsers that have support stats in cssdb.
+export const browsersWithSupportStats = [
+	'and_chr',
+	'and_ff',
+	'android',
+	'chrome',
+	'edge',
+	'firefox',
+	'ie',
+	'ios_saf',
+	'op_mini',
+	'op_mob',
+	'opera',
+	'safari',
+	'samsung',
+];
```

Comment:
If this includes something that we don't have data for, like `baidu` and the browserslist query of the user also includes one of these then the internal logic in `postcss-preset-env` will always transpile everything.

I've moved this to a separate file and I am not using this a filter consistently throughout the codebase.